### PR TITLE
Feature - Winning screen

### PR DESCRIPTION
**Pull request by Terverok**
I introduce hera a winning modal, winners dance, losers cry.

### CODE REVIEW

**Comment by Dammic** [comment]
You should definitely fix those conflicts.

File: src/features/Modal/ModalComponent.jsx [hunk 1]
```diff
@@ -8,8 +8,6 @@ const ModalComponent = ({
     title,
     overlayClosesModal,
     isCloseButtonShown,
-    componentName,
```
Comment (Dammic) [inline src/features/Modal/ModalComponent.jsx]: Why this deletion?

File: src/features/Modal/VotingModal/VotingModal.jsx [hunk 2]
```diff
@@ -2,12 +2,18 @@ import React from 'react'
...
 import { SocketEvents, PlayerRole } from '../../../../Dictionary'
 import { socket } from '../../../utils/SocketHandler'
 import { toggleVotingModal } from '../../../ducks/roomDuck'
 
 export class VotingModal extends React.PureComponent {
+    static propTypes = {
+        data: PropTypes.object,
+        playersDict: PropTypes.object,
+        closeModal: PropTypes.func,
+    };
```
Comment (Dammic) [inline src/features/Modal/VotingModal/VotingModal.jsx]: roomActions is missing

File: src/features/Modal/WinningModal/WinningModal.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+
+    &-list {
+        display: flex;
+        flex-direction: row;
```
Comment (Dammic) [inline src/features/Modal/WinningModal/WinningModal.scss]: 'flex-direction: row' is the default value

File: src/features/PlayerBoard/Player/PlayerAvatar/PlayerAvatar.scss [hunk 4]
```diff
@@ -28,7 +28,6 @@
         &:after {
             transform: rotate(49deg);
         }
-
```
Comment (Dammic) [inline src/features/PlayerBoard/Player/PlayerAvatar/PlayerAvatar.scss]: Why this change?

File: src/utils/SocketHandler.js [hunk 5]
```diff
@@ -110,9 +111,14 @@ export class SocketHandler extends React.PureComponent {
         })
 
         socket.on(SocketEvents.GameFinished, (payload) => {
-            const { isSuccess, facists } = payload.data
+            const { whoWon, facists } = payload.data
             this.props.roomActions.revealFacists(facists)
-            console.info(isSuccess ? 'you won!!' : 'you lose!!!')
+            console.info(whoWon + ' won!');
```
Comment (Dammic) [inline src/utils/SocketHandler.js]: This console info seems to be unnecessary

File: utils/RoomsManager.js [hunk 6]
```diff
@@ -219,6 +219,7 @@ class RoomsManager {
         const nextEmptySlot = freeSlots[0]
 
         if (!nextEmptySlot) {
+
```
Comment (Dammic) [inline utils/RoomsManager.js]: Why this change?

File: src/utils/SocketHandler.js [hunk 7]
```diff
@@ -110,9 +111,14 @@ export class SocketHandler extends React.PureComponent {
         })
 
         socket.on(SocketEvents.GameFinished, (payload) => {
-            const { isSuccess, facists } = payload.data
+            const { whoWon, facists } = payload.data
             this.props.roomActions.revealFacists(facists)
-            console.info(isSuccess ? 'you won!!' : 'you lose!!!')
+            console.info(whoWon + ' won!');
+            this.props.modalActions.setModal({
+                title: 'You won!',
```
Comment (Dammic) [inline src/utils/SocketHandler.js]: When you are a facist, you also get 'You won'. We should make this message different for losers.

**Comment by Dammic** [comment]
This does not always work well - I've come across a case when people who won were in sorrow.
bug

File: src/utils/SocketHandler.js (hunk 5 above)
Comment (Terverok) [reply src/utils/SocketHandler.js]: your's console.log is unnecesssary too
